lightly/utils/scheduler.py

```diff
@@ -144,13 +144,17 @@ class CosineWarmupScheduler(torch.optim.lr_scheduler.LambdaLR):
         max_epochs:
             Total number of training epochs or steps.
         last_epoch:
-            The index of last epoch or step. Default: -1
+            The index of last epoch or step.
         start_value:
-            Starting learning rate scale. Default: 1.0
+            Starting learning rate.
         end_value:
-            Target learning rate scale. Default: 0.001
+            Target learning rate.
         verbose:
-            If True, prints a message to stdout for each update. Default: False.
+            If True, prints a message to stdout for each update.
+        warmup_start_value:
+            Starting learning rate for warmup.
+        warmup_end_value:
+            Target learning rate for warmup. Defaults to start_value.
```

Comment:
Remove defaults message.

Reply:
This is on purpose to let the user know that `warmup_end_value` is set to `start_value` if not specified.